src/Actions/Webhook/DispatchWebhooks.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Cachet\Actions\Webhook;
+
+use Cachet\Enums\WebhookEventEnum;
+use Cachet\Models\WebhookSubscription;
+use Illuminate\Database\Eloquent\Collection;
+
+class DispatchWebhooks
+{
+    protected mixed $event;
+    protected WebhookEventEnum $eventName;
+
+    public function handle(mixed $event): void
+    {
+        $this->event = $event;
+        $this->eventName = $this->event->getWebhookEventName();
+
+        $payload = $this->event->getWebhookPayload();
+
+        foreach ($this->getWebhookSubscriptionsForEvent() as $webhookSubscription) {
+            $webhookSubscription->makeCall($this->eventName, $payload)->dispatch();
+        }
```

Comment:
I wonder if we should offer the ability to configure which queue connection to dispatch webhooks on.